tools/rex/rex.cabal

```diff
@@ -0,0 +1,46 @@
+name:                rex
+version:             0.3.0
+synopsis:            Scrape and expose restund metrics for prometheus
+description:         @rex@ = @restund-exporter@ in prometheus jargon
+author:              Kim Altintop
+maintainer:          kim@wire.com
```

Comment:
this should be changed to Wire Swiss. Also add a license.

Reply:
done